doc/release-notes.md

```diff
@@ -99,6 +99,11 @@ Build System
 Updated settings
 ----------------
 
+- The `-banscore` configuration option, which modified the default threshold for
+  disconnecting and discouraging misbehaving peers, has been removed as part of
+  changes in this release to the handling of misbehaving peers. Refer to the
+  section, "Changes regarding misbehaving peers", for details. (#19464)
+
```

Comment:
Shouldn't this be in `/doc/release-notes-19464.md`? `doc/developer-notes.md` says:
> Release notes should be added to a PR-specific release note file at
`/doc/release-notes-<PR number>.md`

Reply:
See https://github.com/bitcoin/bitcoin/pull/19464#discussion_r452835586